client/src/featureform/resources.py

```diff
@@ -679,7 +734,10 @@ def create_all_local(self) -> None:
     def create_all(self, stub) -> None:
         for resource in self.__create_list:
             try:
-                print("Creating", resource.type(), resource.name)
+                if resource.type()[:3] == "get":
```

Comment:
Whats this doing?

Reply:
checks if the resource is a ProviderReference or EntityReference, etc. (the _create function for those objects are actually getting the object), so the message is different